app/views/admin/basic_projects/show.html.slim

```diff
@@ -26,7 +25,32 @@ section.details
 
     = render 'form'
 
+section.block id="timeline-table"
+  h3 Timeline - Table
+  = render "admin/timeline/main"
+
+div id="timeline-list"
```

Comment:
How content is set up is temporary. Eventually each of these will be a separate page that has tabs at the top taking the user to different pages for a basic project.